src/app/services/views/userOrganisations.ejs

```diff
@@ -14,7 +14,19 @@
 			<div class="govuk-grid-row govuk-!-margin-top-6 govuk-!-margin-bottom-6">
 				<div class="govuk-grid-column-full">
 					<dl class="inline">
-						<dt class="govuk-label"><%=org.name%></dt>
+						<% if(org.urn && org.ukprn) {%>
+							<dt class="govuk-label"><%=org.name%> (URN:<%=org.urn%>,UKPRN:<%=org.ukprn%>)</dt>
+						<%} else {%>
```

Comment:
`<%=org.name%> (URN:<%=org.urn%>,UKPRN:<%=org.ukprn%>)</dt>` we may need to add an extra white space between URN and UKPRN. Please doublecheck with the UI designer.